contracts/src/SubnetRegistryDiamond.sol

```diff
@@ -42,11 +57,17 @@ contract SubnetRegistryDiamond {
         ds.supportedInterfaces[type(IDiamondLoupe).interfaceId] = true;
 
         s.GATEWAY = params.gateway;
-        s.SUBNET_GETTER_FACET = params.getterFacet;
-        s.SUBNET_MANAGER_FACET = params.managerFacet;
+        s.SUBNET_ACTOR_GETTER_FACET = params.getterFacet;
+        s.SUBNET_ACTOR_MANAGER_FACET = params.managerFacet;
+        s.SUBNET_ACTOR_REWARD_FACET = params.rewarderFacet;
+        s.SUBNET_ACTOR_CHECKPOINTING_FACET = params.checkpointerFacet;
+        s.SUBNET_ACTOR_PAUSE_FACET = params.pauserFacet;
```

Comment:
Just noting the different words, `PAUSE` vs `pauser`, `CHECKPOINTING` vs `checkpointer`